Recruit-Training/Embedded-Recruit-Training/Core/Src/main.c

```diff
@@ -288,6 +304,26 @@ void HAL_CAN_RxFifo0MsgPendingCallback(CAN_HandleTypeDef* hcan)
     }
 
     //TODO: Match StdId of header and data length content for green and blue messages and check data for set bit of toggling green and blue led
+    if (hdr.StdId == BLUE_MESSAGE_STDID && hdr.DLC == 1)
+    {
+        uint32_t mask = 10001001;
```

Comment:
I think this ends up being the integer ten million one thousand and one. You want to use 0b10001001
https://stackoverflow.com/a/29487281